forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -6131,6 +6134,26 @@ public void updateReplacementEffects(List<ReplacementEffect> list, CardState sta
         for (KeywordInterface kw : getUnhiddenKeywords(state)) {
             list.addAll(kw.getReplacements());
         }
+
+        // Shield Counter aren't affected by Changed Card Traits
+        if (this.getCounters(CounterEnumType.SHIELD) > 0) {
```

Comment:
Should we change card scripts to `SHIELD` or should this change to `Shield`?

Reply:
the Enum needs to be uppercase, what you do with the scripts is your choice

except from keyword counters, they all use uppercase too